### PR TITLE
Adding example on gravity sensor.

### PR DESCRIPTION
**Pull request by arskama**
During last DAS meeting, it was agreed to merge the existing gravity sensor explainer into motions-sensor explainer.
https://www.w3.org/2021/04/07-dap-minutes.html#r05

link to gravity sensor explainer.
https://github.com/arskama/GravitySensor/blob/main/explainer.md

***
<a href="https://pr-preview.s3.amazonaws.com/arskama/motion-sensors/pull/30.html" title="Last updated on Jun 7, 2021, 10:38 AM UTC (04cb257)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/motion-sensors/30/93d351e...arskama:04cb257.html" title="Last updated on Jun 7, 2021, 10:38 AM UTC (04cb257)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -643,6 +643,48 @@ as the <a>magnetic field</a> vector is more or less stable.
 The [=LinearAccelerationSensor interface=] and [=GravitySensor interface=] are defined in
 [[ACCELEROMETER]] specification.
 
+<div class="example">
+  The following example implements a [=GravitySensor=] polyfill. It shows the complexity of extracting the [=gravity=] value from
+  the [=Accelerometer interface=] and [=LinearAccelerationSensor interface=], in case both sensors are hardware-based sensors.
+  In case there is no hardware-based [=LinearAccelerationSensor=], the web engine will implement some fusion logic based on
+  the [=Accelerometer=] value and provide a less accurate value. In the example, a simplification shortcut has been made
```
Comment (kenchris) [inline index.bs:650]: We say User Agent and not web engine

File: index.bs [hunk 2]
```diff
@@ -643,6 +643,48 @@ as the <a>magnetic field</a> vector is more or less stable.
 The [=LinearAccelerationSensor interface=] and [=GravitySensor interface=] are defined in
 [[ACCELEROMETER]] specification.
 
+<div class="example">
+  The following example implements a [=GravitySensor=] polyfill. It shows the complexity of extracting the [=gravity=] value from
+  the [=Accelerometer interface=] and [=LinearAccelerationSensor interface=], in case both sensors are hardware-based sensors.
+  In case there is no hardware-based [=LinearAccelerationSensor=], the web engine will implement some fusion logic based on
```
Comment (kenchris) [inline index.bs]: will -> needs to implement

File: index.bs [hunk 3]
```diff
@@ -643,6 +643,48 @@ as the <a>magnetic field</a> vector is more or less stable.
...
+  the [=Accelerometer=] value and provide a less accurate value. In the example, a simplification shortcut has been made
+  to listen only to [=accelerometer=] events and not to both sensors' events.
```
Comment (kenchris) [inline index.bs]: to only listen to

File: index.bs [hunk 4]
```diff
@@ -659,6 +701,11 @@ Motion sensors can be used for variety of advanced use-cases, such as:
 - Sensor assisted indoor mapping and navigation [[INDOORNAV]], [[IPIN]]
 - Photo and video capture stabilization [[ROLLSTABILIZER]], [[VIDEOSTABILIZER]]
 - Games that require high-frequency sampling rates with high accuracy readings.
+- Many casual games which do not use touch input but use instead sensor value, such as gravity:
```
Comment (kenchris) [inline index.bs]: but instead use a sensor value

**Review by arskama — COMMENTED** [review]
Fixed on second patch.

**Review by kenchris — APPROVED** [review]

**Pull request merged**